database/database.go

```diff
@@ -0,0 +1,304 @@
+package database
+
+import (
+	"bytes"
+	"context"
+	"fmt"
+	"net/url"
+	"os"
+
+	"github.com/NebulousLabs/skynet-accounts/build"
+	"github.com/NebulousLabs/skynet-accounts/user"
+
+	"gitlab.com/NebulousLabs/errors"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	// EnvDBHost holds the name of the environment variable for DB host.
+	EnvDBHost = "SKYNET_DB_HOST"
```

Comment:
These are exported but not used outside of the `database` package.

They should probably be moved to `main.go` anyway where environment variables are usually read from. Then main calls the constructor with the connection string.

Reply:
I'll unexport them and move them to main in a F/U.

Reply:
F/U: https://github.com/NebulousLabs/skynet-accounts/issues/5